realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/mongodb/sync/SyncedRealmTests.kt

```diff
@@ -554,6 +566,7 @@ class SyncedRealmTests {
                     columnDecimal128 = expectedDecimal128
                     columnObjectId = expectedObjectId
                     columnUUID = expectedUUID
+                    columnMixed = expectedMixed
```

Comment:
It seems a little sparse only checking one of the types, but if we are certain that uploads/download of various changes are verified by the exhaustive test through list, then it is maybe ok. Would have preferred a loop of 
```
columnMixed = expectedTypeA
instance1.uploadAllLocalChanges()
instance2.downloadAllChanges()
assertEquals(....)
````